### ISSUE TITLE
Remove comment highlight in iterm

### ISSUE THREAD

**Issue by ahmedelgabri** [morhetz/gruvbox#44]
I have read the wiki and loaded [`gruvbox_256palette.sh`](https://github.com/morhetz/gruvbox/blob/master/gruvbox_256palette.sh) file in my `.zshrc` and added `g:gruvbox_italic=0` to `.vimrc` and still as you see in the screenshot, comments are still highlighted. Any idea why?
![screen shot 2015-01-25 at 00 38 16](https://cloud.githubusercontent.com/assets/63876/5888951/db371000-a42a-11e4-9594-4e0d81f2f248.png)

**Comment by morhetz** [comment]
Whats you output of `hi Comment`? You should set `let g:gruvbox_italic=0` *before* setting `colorscheme gruvbox`

**Comment by ahmedelgabri** [comment]
Setting `let g:gruvbox_italic=0` before `colorscheme gruvbox` fixed it, thanks!

**Comment by ahmedelgabri** [comment]
@morhetz I have a quick question though. 

I understand that terminal vim color capabilities are not the same as the GUI and sometime it needs the use of shell files to do this. But there was a discussion [here](https://github.com/chriskempson/base16-vim/issues/35) in base-16-vim about this _which also uses the shell method to fix colorization_. But it looks like iTerm2 fixed an issue related to this and now we don't need to source the shell script if you are using iTerm2. Any plans to do the same, make it work without the need of the shell script?

**Comment by morhetz** [comment]
TLDR: Nope.
If I got all things right, base-16 guys are using _terminal_ colorscheme with 16-colorspace, which could be done with the gruvbox the same way.
Set up suitable [iterm theme](https://github.com/morhetz/gruvbox-generalized/tree/master/iterm2) and put that to your .vimrc `let g:gruvbox_termcolors=16`. I'm not really fond of that mostly because this solution forces you to have the same terminal theme _and_ vim theme. It also makes unavailable to switch from gruvbox-dark to gruvbox-light at runtime.
I'm not really aware of base-16 shellscripts, but I suppose they are somehow based on mine.
